package.json

```diff
@@ -70,10 +69,9 @@
     ]
   },
   "resolutions": {
-    "styled-components": "^5",
+    "styled-components": "^6",
     "browserslist": "^4.16.5",
     "ansi-regex": "^5.0.1",
-    "webpack-plugin-ramdisk": "^0.2.0",
     "got": "^11.8.5",
     "@types/react": "^18.2.17"
```

Comment:
Kan være med på at `styled-components` skal ligge her, men skjønner ikke helt hvorfor vi må ha alle de andre her 🤔

Reply:
Typisk for å sikre at transitive avhengigheter er oppdatert mtp sikkerhetsvarsler. Kan være utdatert

Reply:
tror jeg lar de ligge for nå jeg. Syns det er litt skummelt 😅 så kan fikse ved en senere anledning

bare ikke veldig glad i den resolutions-lista, for det var først nå jeg oppdaget at `styled-components` ikke egentlig var oppdatert til v6, fordi man hadde satt v5 som den versjonen man ønsket å bruke